vertical-pod-autoscaler/pkg/recommender/input/controller_fetcher/controller_fetcher_test.go

```diff
@@ -239,17 +272,42 @@ func TestControllerFetcher(t *testing.T) {
 			expectedKey:   nil,
 			expectedError: fmt.Errorf("Cycle detected in ownership chain"),
 		},
+		{
```

Comment:
It should be possible to add a similar case where the parent actually supports scale subresource, WDYT?

Reply:
yeah I shall try that, I wanted to see if you thought this seemed sane before I attempted that :)

Reply:
Thanks! Looks good